src/scan_manager/scan_container.rs

```diff
@@ -446,6 +447,12 @@ impl FeroxScans {
         };
 
         self.menu.clear_screen();
+
+        let config = Configuration::new().unwrap();
+        let target = &config.target_url;
+        let banner = Banner::new(&[String::from(target)], &config);
+        banner.print_to(stderr(), Arc::new(config)).unwrap();
```

Comment:
give this a try, but it should work

```suggestion
        let banner = Banner::new(&[handles.config.target_url.clone()], &handles.config);
        banner
            .print_to(&self.menu.term, handles.config.clone())
            .unwrap_or_default();
```